decorators/uncontrolled.jsx

```diff
@@ -35,7 +35,13 @@ export default ({defaultProp, prop, handlerName, handlerSelector, resetHandlerNa
 
   render () {
     const props = {
-      ...this.props,
+      ...Object.keys(this.props).reduce((filtered, propName) => {
```

Comment:
Also, I can't actually comment on it because Github's reviewing functionality sucks, but down there, you're doing a bind within the render function. That's not a good thing, as it causes you to create new functions on every render, which for a component like this could be _a lot_ of times.